inf-clojure.el

```diff
@@ -986,8 +1016,9 @@ of forms."
             (while (looking-at "\n")
               (delete-char 1))
             (unless (eobp)
-              (clojure-forward-logical-sexp))
-            (unless (eobp)
+              ;; There is no special API for that in
```

Comment:
You removed the `(clojure-forward-logical-sexp)` call but added a comment that you probably keep it in. Comment and action are at odds here.

Reply:
Yep, removed by accident. Will bring it back. Thanks!